openfasoc/common/platforms/sky130hvl/config.mk

```diff
@@ -51,6 +65,11 @@ export CTS_MAX_SLEW   = 1.5e-9
 export CTS_MAX_CAP    = .1532e-12
 export CTS_TECH_DIR   = ../../../common/platforms/$(PLATFORM)/tritonCTShd
 
+export MACRO_PLACE_HALO ?= 1 1
+export MACRO_PLACE_CHANNEL ?= 20 20
+#export MACRO_FENCE_REGION = 250 30 490 170
+export MACRO_PLACEMENT = ../blocks/$(PLATFORM)/manual_macro.tcl
```

Comment:
These attributes should be customized inside the generator config files, not in the common platform

Reply:
Okay. Will change this.